src/core/ledger.js

```diff
@@ -156,4 +157,24 @@ Ledger.prototype.calc_account_hash = function(options) {
   return account_map.hash();
 };
 
+// see rippled Ledger::updateHash()
+Ledger.calculateLedgerHash =
+Ledger.prototype.calculateLedgerHash = function(ledgerHeader) {
+  const so = new SerializedObject();
+  const prefix = 0x4C575200;
+  const totalCoins = (new BigNumber(ledgerHeader.total_coins)).toString(16);
```

Comment:
I think it's more valuable to stick to one bignumber library throughout the whole api directory.